templates/events/_header.html.twig

```diff
@@ -34,12 +34,20 @@
             <div class="text--white text--center">
                 L'événement à été annulé
             </div>
-        {% elseif not event.full%}
-            <a class="btn btn--no-border {{ isFull?' btn--full':''}} register-event" href="{{ path("app_#{event.type}_attend", {slug: event.slug}) }}">Je veux participer</a>
+        {% elseif not event.full %}
```

Comment:
can you refactor this part for writing once the button block, something like that: 
```
{% else %}
     {% if is_event_already_participating(event, app.user) %}
          <button .../>
     {% endif %}

     {% if not event.full %}
           // ...
     {% else %}
           // ...
     {% endif %}
{% endif %}
```